contrib/kind.sh

```diff
@@ -34,6 +34,37 @@ delete()
   kind delete cluster --name ${KIND_CLUSTER_NAME:-ovn}
 }
 
+update_kubeconfig()
+{
+  # For IPv6 NO-HA Shared-Gateway Mode, it is a pure IPv6 network and no IPv4 connectivity.
+  # So need to updae the KubeConfig with an IPv6 address. In HA mode, there is a load-balancer
+  # in front which responds to the IPv4 requests.
```

Comment:
Is IPv6 NO-HA Shared-Gateway the only case where API_URL ends up with a container name in it rather than an IP address? If not, why is it that the container name works in other cases but needs to be rewritten here? If we can resolve the container name to an IP in other cases, why can't we here? (Does the container name resolve to an IPv4 address? Can we fix that?)